### DISCUSSION TITLE
YoungC0DE - Se o pato perde a pata

### DISCUSSION THREAD

**Discussion by Rich14S** [Rich14S/OMG_BADGES#8] [Q&A]
Ele fica manco ou viúvo?

**Comment by YoungC0DE** [comment]


**Reply by Rich14S** [reply]
PARABÉNS! TOMA SUA BADGE!

**Comment by Luciano-Siqueira** [comment]
Talvez manco ou talvez viúvo

**Comment by matheusgasparx** [comment]
Ele fica é rico, se for a muié deixa de gastar e se for o pé, ele se aposenta
